web/worker_service/style.css

```
@@ -1,3 +1,19 @@
+@font-face {
+  font-family: 'TwakeInter';
```

Stylelint (17.14.0)

[error] Expected no quotes around "TwakeInter" (font-family-name-quotes)

(font-family-name-quotes)